application/controllers/admin/CheckIntegrity.php

```diff
@@ -1074,7 +1074,7 @@ protected function checkintegrity()
         $aFullOldTokenSIDs = array();
 
         foreach ($aTables as $sTable) {
-            list($sOldText, $SurveyText, $iSurveyID, $sDateTime) = explode('_', substr($sTable, strlen($sDBPrefix)));
+            list($sOldText, $SurveyText, $iSurveyID, $sDateTime) = explode('_', substr((string) $sTable, strlen((string) $sDBPrefix)));
```

Comment:
Can $sTable be null? What about using `??` to give it a default value when null?

Reply:
Question is, do you want the empty string default in all places?

Reply:
Well, that was the suggested fix by rector.
https://github.com/rectorphp/rector

Reply:
Gah